x-pack/plugins/security/public/authentication/login/components/login_form/login_form.tsx

```diff
@@ -214,27 +214,29 @@ export class LoginForm extends Component<Props, State> {
 
     return (
       <>
-        {this.props.selector.providers.map(provider => (
-          <EuiButton
-            key={provider.name}
-            className="loginWelcome__selectorButton"
-            iconType="user"
-            fullWidth={true}
-            isDisabled={!this.isLoadingState(LoadingStateType.None)}
-            isLoading={this.isLoadingState(LoadingStateType.Selector, provider.name)}
-            onClick={() => this.loginWithSelector(provider.type, provider.name)}
-          >
-            {provider.description ?? (
-              <FormattedMessage
-                id="xpack.security.loginPage.loginProviderDescription"
-                defaultMessage="Login with {providerType}/{providerName}"
-                values={{
-                  providerType: provider.type,
-                  providerName: provider.name,
-                }}
-              />
-            )}
-          </EuiButton>
+        {this.props.selector.providers.map((provider, index) => (
+          <Fragment key={index}>
+            <EuiButton
+              key={provider.name}
+              iconType="user"
```

Comment:
Actually one change request if possible.... Can you remove this icon? It really isn't helpful when all buttons use the same icon.

Reply:
Sure, I can, @ryankeairns is it fine by you as well? Unfortunately we cannot specify different icons yet (but may be able to add this feature in the future).

Reply:
+1 to remove it since we can't differentiate.

Reply:
Thanks @cchaos and @ryankeairns! Done in https://github.com/elastic/kibana/pull/53010/commits/ceba57b46ba5164ca3257c0fe269457a099b903b.